### PR TITLE
added html cheatsheet

### PR DESCRIPTION
**Pull request by SandwichPrestigious**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -186,7 +186,7 @@
 ### HTML and CSS
 
 * [Accessibility CheatSheet](https://learn-the-web.algonquindesign.ca/topics/accessibility-cheat-sheet/) - Algonquin Design (HTML)
-* [Beginner's CheatSheet](https://websitesetup.org/html5-cheat-sheet/)- WebsiteSetup(HTML)
+* [Beginner's CheatSheet](https://websitesetup.org/html5-cheat-sheet/)- WebsiteSetup (HTML)
```
Comment (sohansai) [inline more/free-programming-cheatsheets.md:189]: Missing a space before author

_Corrected citation format:_ 

`[Beginner's CheatSheet](https://websitesetup.org/html5-cheat-sheet) - WebsiteSetup (HTML)`

**Review by SandwichPrestigious — COMMENTED** [review]
added the spaces

**Comment by SandwichPrestigious** [comment]
did the required changes

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**